### PR TITLE
Simplify the CCDB landing page view and template

### PR DESCRIPTION
**Pull request by higs4281**
The CCDB legacy landing view contained a routine for ensuring that
sample complaint narratives and data were available and up-to-date,
and it handled conditions when different banner messages would be displayed. 
The process made a request to Elasticsearch just to get metadata.

This can now be simplified.

Complaint narrative samples are no longer used on the landing page and
don't need to be checked. In addition, the check for stale data never worked,
because of a bug. The view code was checking for a nonexistent [response.status](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/legacy/views/complaint.py#L48) value
instead of `response.status_code`, and it always returned blank json.

Data for the page's high-charts map and metadata for staleness were moved to S3,
where they are updated every morning. Those resources are called from the page.

All this allows us to stop making view calls to Elasticsearch entirely.
The view just needs to deliver the news when the CCDB_TECHNICAL_ISSUES flag is
activated, and that's all we need to test now.

## PR testing
- Pull in the branch and check that the CCDB landing page renders locally
at http://localhost:8000/data-research/consumer-complaints/
- in the local Wagtail admin, go to /admin/flags/ and enable the flag
` CCDB_TECHNICAL_ISSUES`
- The landing page should now show a warning dialog below the hero:

ccdb_warning

### CODE REVIEW

**Comment by flacoman91** [comment]
how does CCDB_TECHNICAL_ISSUES get changed? is it automatic, or is someone supposed to manually set it in wagtail?

**Comment by higs4281** [comment]
@flacoman91 `CCDB_TECHNICAL_ISSUES` is a flag that can be set in Wagtail if a problem is discovered.

File: cfgov/jinja2/v1/complaint/complaint-landing.html [hunk 1]
```diff
@@ -68,22 +68,16 @@ <h1 class="m-hero_heading">Consumer Complaint Database</h1>
 
 {% block content_main %}
 
-<div class="m-notification m-notification__warning {% if data_down %} m-notification__visible show-data-notification {% elif narratives_down %} m-notification__visible show-narratives-notification {% endif %}">
+<div class="m-notification m-notification__warning {% if technical_issues %} m-notification__visible show-data-notification {% endif %}">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/complaint/complaint-landing.html:71]: If this is a straight-up static notification that is shown/hidden it looks like we can get rid of the `show-data-notification` class here, `data-text` class on line 76, and this CSS https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/ccdb-landing-map/css/landing-page.less#L63-L80

Comment (higs4281) [reply cfgov/jinja2/v1/complaint/complaint-landing.html:71]: √

**Comment by flacoman91** [comment]
Looks good. I was able to get this to work.

**Review by flacoman91 — APPROVED** [review]
👍

File: cfgov/jinja2/v1/complaint/complaint-landing.html [hunk 2]
```diff
@@ -68,12 +68,13 @@ <h1 class="m-hero_heading">Consumer Complaint Database</h1>
 
 {% block content_main %}
 
-<div class="m-notification m-notification__warning {% if technical_issues %} m-notification__visible show-data-notification {% endif %}">
+{% if technical_issues %}
+<div class="m-notification m-notification__warning m-notification__visible ">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/complaint/complaint-landing.html:72]: TRIVIAL: could be remove the ending whitespace after the end of the classes `m-notification__visible">`

Comment (higs4281) [reply cfgov/jinja2/v1/complaint/complaint-landing.html]: √

**Comment by Scotchester** [comment]
<obligatory drive-by this-should-be-a-Wagtail-page comment>

**Comment by higs4281** [comment]
@Scotchester `<obligatory true-that>`

**Pull request merged**